source/game/kart/KartPhysics.hh

```diff
@@ -39,6 +48,7 @@ private:
     EGG::Vector3f m_yAxis;
     EGG::Vector3f m_zAxis;
     EGG::Vector3f m_velocity;
+    f32 m_fc; // collisionLimit?
```

Comment:
`computeCollisionLimits` returns `0.5 * max.z` where `max` is the largest out of all of the hitboxes `radius + position`. I'm not sure the relevance of `0.5` but this pertains to scaling the position used in collision function checks. It's only used in `calcTriggers`. Not immediately sure what to call this